packages/getters/src/swap-plaintext.ts

```diff
@@ -1,6 +1,11 @@
-import { SwapPlaintext } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/dex/v1/dex_pb';
+import {
+  SwapBody,
+  SwapPlaintext,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/dex/v1/dex_pb';
 import { createGetter } from './utils/create-getter';
 
 export const getDelta1I = createGetter((swapPlaintext?: SwapPlaintext) => swapPlaintext?.delta1I);
 export const getDelta2I = createGetter((swapPlaintext?: SwapPlaintext) => swapPlaintext?.delta2I);
 export const getClaimFee = createGetter((swapPlaintext?: SwapPlaintext) => swapPlaintext?.claimFee);
+export const getDelta1IOpaque = createGetter((swapBody?: SwapBody) => swapBody?.delta1I);
+export const getDelta2IOpaque = createGetter((swapBody?: SwapBody) => swapBody?.delta2I);
```

Comment:
Getter inputs and the file names should match. This means these should move to a file like:

```
packages/getters/src/swap-body.ts
```

Reply:
Also, we should remove `Opaque` from the name

Reply:
moved to dedicated file